armi/reactor/converters/tests/test_axialExpansionChanger.py

```diff
@@ -662,34 +662,19 @@ def test_determineTargetComponentRuntimeErrorSecond(self):
             self.assertEqual(the_exception.error_code, 3)
 
     def test_isFuelLocked(self):
```

Comment:
Please add a docstring to this test to explain what it does and how it works.

Reply:
https://github.com/terrapower/armi/pull/1038/commits/b940f8bb322b970b0caeafca7da4c56e809f8a49

Reply:
Extend to say something like:

```
"""
Ensures that the RuntimeError statement in ExpansionData::_isFuelLocked is raised appropriately.

Notes
------
This is implemented by creating a fuel block that contains no fuel component. 
"""
```

Reply:
https://github.com/terrapower/armi/pull/1038/commits/a4a561f26b84d2ae7dc661017a3168636bdb305f